phoneApp/js/keypad.js

```diff
@@ -0,0 +1,69 @@
+class KeypadPage {
+    constructor () {
+        this.keypad = ['1','2','3','4','5','6','7','8','9','*','0','#']
+
+    }
+    render() {
+        const mountNode = document.createElement('div');
+        mountNode.setAttribute('id', 'mountNode');
+        mountNode.innerHTML += this.renderHeader() +this.renderMain() + this.renderFooter();
```

Comment:
you know... if we just could add "html-layout" without method dividings it could be more informative that 
renderHeader + remderMain + renderFooter